.github/workflows/main.yml

```
@@ -0,0 +1,40 @@
+name: TerminalImageViewer
+
+on:
+  push:
+    branches:
+    - '*'
+    paths-ignore:
+      - '**/README.md'
+  pull_request:
+    branches:
+    - '*'
+  workflow_dispatch:
+
+jobs:
+  image:
+    name: build ${{ matrix.arch_name }} ${{ matrix.build_type }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        #Disable: windows-static-x64, windows-static-x64-posix, windows-static-x86, 
+        # windows-shared-x64, windows-shared-x64-posix, windows-shared-x86, web-wasm, 
+        arch_name: [android-arm, android-arm64, android-x86, android-x86_64, 
+          linux-arm64, linux-arm64-musl, linux-armv5, linux-armv5-musl, 
+          linux-armv6, linux-armv6-lts, linux-armv6-musl, linux-arm64-full, 
+          linux-armv7, linux-armv7a, linux-armv7l-musl, 
+          linux-x64-clang, linux-s390x, linux-x64, linux-x86, 
+          linux-mips, linux-ppc64le, windows-armv7, 
+          manylinux2014-x64, manylinux2014-x86, windows-arm64, 
+          linux-riscv64, linux-riscv32, linux-m68k-uclibc, linux-xtensa-uclibc, linux-x64-tinycc]
```

Why add so much linux distros yet only add windows ARM, which won't work, and not mention macOS at all? I'm also pretty sure that most of those aren't valid github actions OSes... "arch_name" is also a kinda weird name for a matrix...
```suggestion
        arch_name: [12-arm64, 12, ubuntu-latest]
```